fall18/p3/main.py

```diff
@@ -0,0 +1,89 @@
+import project 
+
+# my-login: YOUR-NETID
+# partner-login: PARTNER-NETID
+
+#Function 1
+#testGetNumRecords()
+def testGetNumRecords():
+    pass
+
+
+
+#Function 2
+#testGetName(index)
+
+
+#Function 3
+#testGetWindSpeed(index)
+
+#Function 4
+#searchHurricane(name)
+def serachHurricane(name):
+    #create counter here
+    while(False): #stopping condition, this will currently not loop, fix it
+        #code you want to occur every loop goes here     
+        pass
+        #code you want to occur every loop stops here
+        #increment should go here
+    return 0 
+
+#Function 5
+#searchHurricaneByLocation(lat, long)
+
+
+
+#Function 6
+#countHurricane(oceanName)       
+        
+
+#Function 7
+#maxWindSpeedHurricane()
+
+#Function 8 
+#minWindSpeedHurricane()
+
+def main():
+    print("Problem 1")
+    print("The number of records in the DataSet is ")
+    print(0)
+    print()
+    
+    print("Problem 2")
+    print("The name of the hurricane in the 20th row of the dataset is ")
+    print(0)
+    print()
+    
+    print("Problem 3")
+    print("Does the hurricane 'BOB' Exist in the Dataset?")
+    print(True)

```

Comment:
The test passes for problem 3. Was this intended? Some students might think that we already did this for them! :)

Maybe we can change this to False?